### ISSUE TITLE
Text wrapping in table cells

### ISSUE THREAD

**Issue by adidahiya** [palantir/blueprint#531]
I'd like to be able to set a new option on table columns which makes its cells wrap their text content (rather than putting it on one line and truncating). When enabled, I expect all rows to expand to fill as much space is necessary to display all the text of that newly text-wrapped column.

cc @helenkg

**Comment by tgreenwatts** [comment]
Beta feedback: "Can't read full value of “long” strings in table - e.g. 'user-agent' strings in web logs - I want to look at all of them to compare, and the comparable details are at the end of the string so cut off by the preview. Clicking the ellipsis to expand row-by-row is untenable."

**Comment by michael-yx-wu** [comment]
could we make this a toggle -able option in the table top-left menu? we currently don't have any use for the menu there but this feature seems like a good candidate. we can additionally add the following to table props & state:

props:
```
/** 
 * if true, the table will initially render with content wrapped. content wrapping can be toggled 
 * on or off in the table menu
 * @default false
 */
wrapContentsByDefault: boolean;
```

state:
```
/** 
 * if true, wrap contents instead of truncating 
 * @default false
 */
wrapContents: boolean;
```

**Comment by tgreenwatts** [comment]
I think this is a large enough change we need to spend some serious time thinking about it first, so we'll be doing that next week and then discuss implementation details

**Comment by adidahiya** [comment]
Fixed by #809
